packer/instance-scripts/init-bootnode.sh

```diff
@@ -79,15 +72,30 @@ wait_for_successful_command 'vault status'
 # Wait for vault to be fully configured by the root user
 wait_for_successful_command 'vault auth -method=aws'
 
-wait_for_terraform_provisioners
-
 # Get metadata for this instance
 INDEX=$(cat /opt/quorum/info/index.txt)
 AWS_REGION=$(cat /opt/quorum/info/aws-region.txt)
-PUBLIC_IP=$(wait_for_successful_command 'curl http://169.254.169.254/latest/meta-data/public-ipv4')
+PUBLIC_IP=$(cat /opt/quorum/info/public-ip.txt)
+USING_EIP=$(cat /opt/quorum/info/using-eip.txt)
+EIP_ID=$(cat /opt/quorum/info/eip-id.txt)
+INSTANCE_ID=$(wait_for_successful_command 'curl -s http://169.254.169.254/latest/meta-data/instance-id')
 HOSTNAME=$(wait_for_successful_command 'curl http://169.254.169.254/latest/meta-data/public-hostname')
 BOOT_PORT=30301
 
+# Associate the EIP with this instance
+echo ">> Value of USING_EIP follows on next line:"
+echo ">> $USING_EIP"
+echo ">> (value printed)"
+if [ "$USING_EIP" == "1" ]
+then
+    echo ">> Found USING_EIP == 1, running aws ec2 associate-address"
+    wait_for_successful_command "aws ec2 associate-address --instance-id $INSTANCE_ID --allocation-id $EIP_ID --region $AWS_REGION --allow-reassociation"
+else
```

Comment:
Maybe make this an `elif` statement that checks for 0 so if and when they break this behavior we can fail loudly in the else block?